addons/project/models/project_task.py

```diff
@@ -299,6 +299,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain):
             ! Set the task a high priority\n
             Make sure to use the right format and order e.g. Improve the configuration screen #feature #v16 @Mitchell !""",
     )
+    link_preview_name = fields.Char(compute='_compute_link_preview_name', help="Name used in the link preview")
```

Comment:
It is more a technical field, no? If yes, then we don't really need to add an helper message, IMO.

Also, we can also avoid translating the string for this techincal field by adding `export_string_translation=False` inside the field definition. 🤷🏼‍♂️
(except if we display the label of this field somewhere) 

```suggestion
    link_preview_name = fields.Char(compute='_compute_link_preview_name', export_string_translation=False)
```